src/intercept.c

```diff
@@ -30,11 +30,11 @@ setup_intercepts()
         = cheri_address_set(cheri_pcc_get(), (uintptr_t) comp_exec_out);
 }
 
-size_t
-my_call_comp(
-    size_t comp_id, char *fn_name, void *args) // TODO , size_t args_count)
-{
-    struct Compartment *to_call = manager_get_compartment_by_id(comp_id);
-    return exec_comp(to_call, fn_name, args);
-    /*return exec_comp(to_call, fn_name, args, args_count);*/
-}
+/*size_t*/
```

Comment:
Commented code.

Reply:
This is a leftover `TODO` from when we moved from static binaries, and I broke inter-compartment calls. I added a new `TODO` comment to explain in 380bcc3.